libr/core/panels.c

```diff
@@ -2589,8 +2589,8 @@ void __move_panel_to_down(RCore *core, RPanel *panel, int src) {
 	int i = 0;
 	for (; i < panels->n_panels - 1; i++) {
 		RPanel *tmp = __get_panel (panels, i);
-		int t_y = ((double)tmp->view->pos.y / (double)h) * (double)new_h + 1;
-		int t_h = ((double)tmp->view->pos.h / (double)h) * (double)new_h + 1;
+		const size_t t_y = ((double)tmp->view->pos.y / (double)h) * (double)new_h + 1;
+		const size_t t_h = (tmp->view->edge & (1 << PANEL_EDGE_BOTTOM)) ?  new_h - t_y : ((double)tmp->view->pos.h / (double)h) * (double)new_h;
```

Comment:
if you multiply before dividing you can probably drop the double casts and make the line more readable and avoid depending on the fpu